src/codeflare_sdk/common/utils/unit_test_support.py

```diff
@@ -498,7 +498,11 @@ def create_cluster_all_config_params(mocker, cluster_name, is_appwrapper) -> Clu
         extended_resource_mapping={"example.com/gpu": "GPU", "intel.com/gpu": "TPU"},
         overwrite_default_resource_mapping=True,
         local_queue="local-queue-default",
-        annotations={"key1": "value1", "key2": "value2"},
+        annotations={
+            "app.kubernetes.io/managed-by": "test-prefix",
```

Comment:
```suggestion
```
This one should not be set manually as it is determined by an environment variable